docs/src/contributors.ts

```diff
@@ -0,0 +1,83 @@
+import contributors from '../contributors.json'
+
+export interface Contributor {
+  name: string
+  avatar: string
+}
+
+export interface CoreTeam {
+  avatar: string
+  name: string
+  github: string
+  twitter?: string
+  sponsors: boolean
+  description: string
+}
+
+// for antfu sponsors
+const jsdelivr = 'cdn.jsdelivr.net'
+// for patak sponsors
+const patak = 'patak.dev'
+const antfuSponsors = `https://${jsdelivr}/gh/antfu/static/sponsors.svg`
+const patakSponsors = `https://${patak}/sponsors.svg`
+
+const contributorsAvatars: Record<string, string> = {}
+
+const contributorList = (contributors as string[]).reduce((acc, name) => {
+  contributorsAvatars[name] = `https://github.com/${name}.png`
+  acc.push({ name, avatar: contributorsAvatars[name] })
+  return acc
+}, [] as Contributor[])
+
+const coreTeamMembers: CoreTeam[] = [
+  {
+    avatar: contributorsAvatars.antfu,
+    name: 'Anthony Fu',
+    github: 'antfu',
+    twitter: 'antfu7',
+    sponsors: true,
+    description: 'A fanatical open sourceror<br>Core team member of Vite & Vue<br>Working at NuxtLabs',
+  },
+  {
+    avatar: contributorsAvatars['patak-dev'],
+    name: 'Patak',
+    github: 'patak-dev',
+    twitter: 'patak-dev',
+    sponsors: true,
+    description: 'A collaborative being<br>Core team member of Vite<br>Team member of Vue',
+  },
+  {
+    avatar: contributorsAvatars.Aslemammad,
+    name: 'Aslemammad',
```

Comment:
That's amazing, Could you make my name here `Mohammad Bagher` ?

Reply:
Change it yourself here editing the file and commiting changes to the brabch

Reply:
If you dont want to change yourself just wait tomorrow or maybe @patak-dev can change it before merging into main